PS C:\Products\projects\python> ^C
PS C:\Products\projects\python> & c:/Products/projects/python/venv/Scripts/python.exe c:/Products/projects/python/database_operations/runner.py
Database connection established.
Cursor closed.
Connection closed.
Schema mismatched
INSERT INTO TEST0001 (ID, NAME, PHONE, EMAIL) VALUES (1, 'RED', '353XXXX', 'red@test.com'); INSERT INTO TEST0001 (ID, NAME, PHONE, EMAIL) VALUES (2, 'GREEN', '353XXXY', 'green@text.com');
Database connection established.
Traceback (most recent call last):
  File "c:\Products\projects\python\database_operations\runner.py", line 22, in execute_sql_query
    cursor.execute(i)
    ~~~~~~~~~~~~~~^^^
  File "C:\Products\projects\python\venv\Lib\site-packages\teradatasql\__init__.py", line 752, in execute
    self.executemany (sOperation, None, ignoreErrors)
    ~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Products\projects\python\venv\Lib\site-packages\teradatasql\__init__.py", line 1007, in executemany
    raise OperationalError (sErr)
teradatasql.OperationalError: [Version 20.0.0.21] [Session 2416788] [Teradata Database] [Error 3706] Syntax error: expected something between the beginning of the request and the word 'I'.
 at gosqldriver/teradatasql.formatError ErrorUtil.go:92
 at gosqldriver/teradatasql.(*teradataConnection).formatDatabaseError ErrorUtil.go:252
 at gosqldriver/teradatasql.(*teradataConnection).makeChainedDatabaseError ErrorUtil.go:268
 at gosqldriver/teradatasql.(*teradataConnection).processErrorParcel TeradataConnection.go:751
 at gosqldriver/teradatasql.(*TeradataRows).processResponseBundle TeradataRows.go:2308
 at gosqldriver/teradatasql.(*TeradataRows).executeSQLRequest TeradataRows.go:874
 at gosqldriver/teradatasql.newTeradataRows TeradataRows.go:720
 at gosqldriver/teradatasql.(*teradataStatement).QueryContext TeradataStatement.go:122
 at gosqldriver/teradatasql.(*teradataConnection).QueryContext TeradataConnection.go:1261
 at database/sql.ctxDriverQuery ctxutil.go:48
 at database/sql.(*DB).queryDC.func1 sql.go:1776
 at database/sql.withLock sql.go:3530
 at database/sql.(*DB).queryDC sql.go:1771
 at database/sql.(*Conn).QueryContext sql.go:2027
 at main.createRows goside.go:1080
 at main.goCreateRows goside.go:959
 at _cgoexp_e3ee842aae7c_goCreateRows _cgo_gotypes.go:414
 at runtime.cgocallbackg1 cgocall.go:403
 at runtime.cgocallbackg cgocall.go:322
 at runtime.cgocallback asm_amd64.s:1079
 at runtime.goexit asm_amd64.s:1695

[Version 20.0.0.21] [Session 2416788] [Teradata Database] [Error 3706] Syntax error: expected something between the beginning of the request and the word 'I'.
 at gosqldriver/teradatasql.formatError ErrorUtil.go:92
 at gosqldriver/teradatasql.(*teradataConnection).formatDatabaseError ErrorUtil.go:252
 at gosqldriver/teradatasql.(*teradataConnection).makeChainedDatabaseError ErrorUtil.go:268
 at gosqldriver/teradatasql.(*teradataConnection).processErrorParcel TeradataConnection.go:751
 at gosqldriver/teradatasql.(*TeradataRows).processResponseBundle TeradataRows.go:2308
 at gosqldriver/teradatasql.(*TeradataRows).executeSQLRequest TeradataRows.go:874
 at gosqldriver/teradatasql.newTeradataRows TeradataRows.go:720
 at gosqldriver/teradatasql.(*teradataStatement).QueryContext TeradataStatement.go:122
 at gosqldriver/teradatasql.(*teradataConnection).QueryContext TeradataConnection.go:1261
 at database/sql.ctxDriverQuery ctxutil.go:48
 at database/sql.(*DB).queryDC.func1 sql.go:1776
 at database/sql.withLock sql.go:3530
 at database/sql.(*DB).queryDC sql.go:1771
 at database/sql.(*Conn).QueryContext sql.go:2027
 at main.createRows goside.go:1080
 at main.goCreateRows goside.go:959
 at _cgoexp_e3ee842aae7c_goCreateRows _cgo_gotypes.go:414
 at runtime.cgocallbackg1 cgocall.go:403
 at runtime.cgocallbackg cgocall.go:322
 at runtime.cgocallback asm_amd64.s:1079
 at runtime.goexit asm_amd64.s:1695
Cursor closed.
Connection closed.
PS C:\Products\projects\python> 
